tests/test_instantiate_config.py

```diff
@@ -44,6 +45,7 @@ class TestInstantiateConfig(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
+        [CoreLib.cache_registry.unregister(key) for key in CoreLib.cache_registry.registered()]

```

Comment:
please use here the utility `def sync_create_core_lib_config(path: str, config_file: str = 'config.yaml'):`